com.visualstudio.code.yaml

```diff
@@ -23,6 +23,7 @@ finish-args:
   - --env=NPM_CONFIG_GLOBALCONFIG=/app/etc/npmrc
   - --env=LD_LIBRARY_PATH=/app/lib
   - --filesystem=xdg-config/kdeglobals:ro
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
```

Comment:
```suggestion
  - --filesystem=xdg-run/app/com.discordapp.Discord:create
  - --filesystem=xdg-run/app/com.discordapp.DiscordCanary:create
```

Reply:
> Supporting Discord Canary would be nice too.

Where is there a Flatpak for Discord Canary?

Reply:
https://github.com/flathub/com.discordapp.DiscordCanary, only in the `flathub-beta` repo.